src/lumigo_tracer/event/event_trigger.py

```diff
@@ -118,6 +120,23 @@ def _parse_sns(event: dict):
     }
 
 
+def _is_event_bridge(event: dict):
+    return (
+        "version" in event
+        and "id" in event  # noqa: W503
+        and "detail-type" in event  # noqa: W503
+        and "source" in event  # noqa: W503
+        and "time" in event  # noqa: W503
+        and "region" in event  # noqa: W503
+        and "resources" in event  # noqa: W503
+        and "detail" in event  # noqa: W503
```

Comment:
Looks too generic, try to find something deeper that related to the event bridge

Reply:
Unfortunately from the event size we have nothing..
Maybe I can check also the types of these fields, but the was infro doesn't give me anything...
This is how it looks:
![image](https://user-images.githubusercontent.com/47108628/96607240-d4707680-1300-11eb-845f-b6a7b3ff5d5c.png)

Reply:
wdyt? @doriaviram